### ISSUE TITLE
import path conflicts with up to date package versions

### ISSUE THREAD

**Issue by labaxter** [jakemac53/import_path#1]
I thought this might be help me find where my code is importing dart:mirror. I followed brief installation instructions, but no joy:

```
$ dart pub add import_path
Because build_runner >=2.1.6 depends on analyzer >=1.4.0 <4.0.0 and every version of import_path depends on analyzer ^0.39.0, build_runner >=2.1.6 is incompatible with import_path.
So, because vault depends on both import_path any and build_runner ^2.1.7, version solving failed.
```
So I temporarily removed build_runner from my pubspec.  Now it complains about json_serializable.

```
dart pub add import_path
Because json_serializable >=6.1.3 depends on analyzer >=2.0.0 <4.0.0 and every version of import_path depends on analyzer ^0.39.0, json_serializable >=6.1.3 is incompatible with import_path.
So, because vault depends on both import_path any and json_serializable ^6.1.4, version solving failed.

```
I think this needs to be updated to use current package versions.

**Comment by jakemac53** [comment]
@labaxter you should globally activate this package instead `dart pub global activate import_path`. But I can get on updating the dependencies as well :).

**Comment by jakemac53** [comment]
Ok, I just published version `1.0.2` with updated deps as well 👍
